test/notifications/display_test.dart

```diff
@@ -208,6 +211,158 @@ void main() {
         ..vibrationPattern.isNotNull().deepEquals(
             NotificationChannelManager.kVibrationPattern);
     });
+
+    test('on Android 28 (and lower) resource file is used for notification sound', () async {
+      addTearDown(testBinding.reset);
+      final androidNotificationHost = testBinding.androidNotificationHost;
+
+      // Override android version
+      testBinding.deviceInfoResult =
+        const AndroidDeviceInfo(sdkInt: 28, release: '10');
```

Comment:
nit:
```suggestion
      testBinding.deviceInfoResult =
        const AndroidDeviceInfo(sdkInt: 28, release: '10');
```

The content of this comment is already expressed clearly enough by the code just below it.